crates/parser/tests/snapshots/statements/valid/advisory_lock@13.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/parser/tests/statement_parser_test.rs
+description: ROLLBACK;
+---
+Parse {
+    cst: SourceFile@0..9
+      Rollback@0..8 "ROLLBACK"
+      Ascii59@8..9 ";"
+    ,
+    errors: [],
+    stmts: [],
```

Comment:
`rollback` statement is not part of the AST

https://www.postgresql.org/docs/current/parser-stage.html#PARSER-STAGE-TRANSFORMATION-PROCESS